gateway/app/schemas/qodProvisioning.py

```diff
@@ -0,0 +1,226 @@
+from __future__ import annotations
+
+from datetime import datetime
+from enum import Enum
+from ipaddress import IPv4Address, IPv6Address
+from typing import Any, Dict, Optional, Union
+from uuid import UUID
+
+from pydantic import AnyUrl, BaseModel, Field
+
+
+class ProvisioningId(BaseModel):
+    __root__: UUID = Field(description="Provisioning Identifier in UUID format")
```

Comment:
Don't use `__root__`

```suggestion
ProvisioningId = Annotated[UUID, Field(description="Provisioning Identifier in UUID format")]
```